### PR TITLE
Add movefuns dao work.

### PR DESCRIPTION
**Pull request by peterwong3000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ Move is a programming language for writing safe smart contracts originally devel
 - [Move @ Starcoin Discord](https://discord.gg/starcoin)
 - [Move @ Aptos Discord](https://discord.gg/aptoslabs)
 - [MoveChina](https://move-china.com) - The largest Chinese community for the Move programming language.
+- [Movefuns](https://github.com/movefuns) - The main goal of uniting the developer community to build a across multiple Move chains ecosystem.
```
Comment (jolestar) [inline README.md]:
```
[MoveFuns](https://movefuns.org)
```

**Comment by jolestar** [comment]
Help add the https://github.com/movefuns/move-js and https://github.com/movefuns/move-wasm to `Miscellaneous` ?

**Comment by peterwong3000** [comment]
> Help add the https://github.com/movefuns/move-js and https://github.com/movefuns/move-wasm to `Miscellaneous` ?


Sure. so can you review again?  please

**Comment by jolestar** [comment]
> > Help add the https://github.com/movefuns/move-js and https://github.com/movefuns/move-wasm to `Miscellaneous` ?
> 
> Sure. so can you review again? please

Need @sblackshear to approve

**Pull request closed without merging**